### PR TITLE
Sql swagger descriptions

### PR DESCRIPTION
**Pull request by mmacy**
**Documentation updates only**: Description field edits only in this PR.

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] Validation errors from the [Linter extension for VS Code](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/linter.md) have all been fixed for this spec. (**Note:** for large, previously checked in specs, there will likely be many errors shown. Please contact our team so we can set a timeframe for fixing these errors if your PR is not going to address them).
  - [ ] The spec follows the patterns described in the [Swagger good patterns](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md) document unless the service API makes this impossible.

---
This change is [Reviewable](https://reviewable.io/reviews/azure/azure-rest-api-specs/715)

### CODE REVIEW

**Review by BryanLa — COMMENTED** [review]
@mmacy - looks great. LMK if you want to make any changes and I can get this merged for you.

**Comment by BryanLa** [comment]
Closing/reopening to see if CI will trigger and complete successfully

File: arm-sql/2014-04-01/swagger/sql.core.json [hunk 1]
```diff
@@ -1477,7 +1477,7 @@
         "dtu": {
           "type": "number",
           "format": "double",
-          "description": "Gets or sets the DTUs (Database Transaction Units).  See https://azure.microsoft.com/en-us/documentation/articles/sql-database-what-is-a-dtu/"
+          "description": "Gets or sets the DTUs (Database Transaction Units). See https://azure.microsoft.com/en-us/documentation/articles/sql-database-what-is-a-dtu/"
```
Comment (mmacy) [inline arm-sql/2014-04-01/swagger/sql.core.json]: @BryanLa Yes, I will update. Please stand by.

**Comment by BryanLa** [comment]
@amarzavery - any thoughts on why the CI is failing? Or should I take that up with Sanketh?

**Comment by amarzavery** [comment]
@BryanLa - investigating.

**Comment by amarzavery** [comment]
image

This is the difference.

In one of the specs, the description for "id" property of the "Resource" model is different than the other. Since we are packaging these individual specs together into one REST Client we are using the composite model. Autorest throws if there is a semantic difference between the two types and also throws when there is single character difference in the description of models and its properties between those specs.

**Comment by BryanLa** [comment]
Thanks at ton, @amarzavery. 
@mmacy - FYI. Looks like the Swagger spec needs a few more changes.

**Comment by mmacy** [comment]
@BryanLa @amarzavery On it! Stand by.

**Comment by mmacy** [comment]
@BryanLa @amarzavery OK, all checks passed, I think we're good on this one. Thanks for the debug, Amar!

**Comment by mmacy** [comment]
@sankethka Am I too late on this? Would love a merge.

**Comment by BryanLa** [comment]
@mmacy - I don't think Sanketh is doing the final merge until Sunday evening, so I will go ahead and merge.

**Pull request merged**